scripts/cross_compile_pp_win.py

```
@@ -0,0 +1,76 @@
+    + ["-I" + p for p in inc_dirs]
+    + ["-Fu" + p for p in unit_dirs]
+    + ["--pp-cache-dir=/tmp/win_cache_pp3"]
+)
+
+cmd = [os.path.join(REPO, "build/KGPC/kgpc"),
+       os.path.join(FPC, "compiler", "pp.pas"),
+       "/tmp/pp_win.s", "--emit-link-args"] + flags
+
+print("CMD prefix:", " ".join(cmd[:4]), "... (+%d flags)" % len(flags), flush=True)
+
+r = subprocess.run(cmd, capture_output=True, text=True, timeout=900)
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)